### DISCUSSION TITLE
Забанен игрок без причины

### DISCUSSION THREAD

**Discussion by gat3v** [Sokol-Games-GmbH/PixelCombats.PublicApi#245] [Show and tell]
Игрока недавно забанили твинк и не может зайти на свой аккаунт не знаю за что именно забанили, но данный игрок не использовал Читы или что-то другое запрещённое.

Требую разбанить игрока с айди 63844748E70911D7, тк нету и не была причин для бана

**Comment by kkohno** [comment]
могли читеры забанить, но это не на долго - там разбанится быстро
